docs/getting-started/github-apps/migrating-tools-to-github-apps.rst

```diff
@@ -0,0 +1,158 @@
+you want to migrate has a tool name, select the tool and look at the title on top as shown in the picture below.
+
+The title consists of:
+``<image registry>/<organization name>/<repository name>/<optional tool name>:<version name>``
+
+If you see a tool name inserted, you must include the name field in your ``/.dockstore.yml``.
+
+.. code:: yaml
+
+   version: 1.2
+   tools:
+      - subclass: CWL
+        primaryDescriptorPath: /Dockstore.cwl
+        testParameterFiles:
+            - /test.json
+        name: optional-name
+
+If you have multiple tools registered on Dockstore that stem from the same GitHub repo, a single ``/.dockstore.yml`` can be used to convert them.
+Again, you need to check for the ``Tool Name`` field being set because it's needed for multi tool repositories to ensure that the tool paths on Dockstore are unique.
+
+Let's say we want to convert these two tools that come from this `repository <https://github.com/dockstore/dockstore-tool-bamstats>`__.
+
+.. image:: /assets/images/docs/github-apps-multiple-tools.png
+
+.. image:: /assets/images/docs/github-apps-multiple-tools-with-name.png
+
+
+Your ``/.dockstore.yml`` would look like the following:
+
+.. code:: yaml
+
+   version: 1.2
+   tools:
+      - subclass: CWL
+        primaryDescriptorPath: /Dockstore.cwl
+        testParameterFiles:
+            - /test.cwl.json
+      - subclass: CWL
+        primaryDescriptorPath: /Dockstore2.cwl
+        testParameterFiles:
+            - /test.cwl.json
+        name: optional-name
+
+Deleting Your Existing Tool
+---------------------------
+
+Once you've adding a ``.dockstore.yml`` to the desired branch of your repository, you should see a new tool appear on your ``/my-tools`` page. The tool path will start with ``github.com`` and 
+you should see that the ``Tool Information`` section looks a bit different from your existing tool.
+
+.. figure:: /assets/images/docs/github-app-tool.png
+   :alt: New GitHub App tool
+
+The mode is ``Automatically synced via GitHub App`` instead of one of our three build modes, and information about paths and your Docker Image is no longer included.
+You are also not able to refresh or restub the new GitHub App tool. Since you can't refresh the entire tool anymore, **new** versions from GitHub (releases/branches) that you want to add to Dockstore must have a ``.dockstore.yml`` file.
+
+If you look on the left hand side bar, you should see that your old tool still exists. Your new GitHub App tool will start with ``github.com``, and your old tool will start with
+the Docker registry, which is ``quay.io`` in this case.
+
+.. figure:: /assets/images/docs/old-and-new-tool.png
+   :alt: Old and new tools
+   :width: 50 %
+
+At this point, you must decide whether you want to keep your old tool around. You may want to keep it if you want to preserve the link to the tool. For example, you may choose to keep the tool if this tool was linked in a research paper
```

Comment:
Ok, it's good that its here but the note earlier could be helpful too if people are skimming.